Composer/packages/client/src/pages/language-generation/table-view.tsx

```diff
@@ -329,7 +329,7 @@ const TableView: React.FC<TableViewProps> = (props) => {
           iconProps={{ iconName: 'CirclePlus' }}
           onClick={() => {
             onCreateNewTemplate();
-            setMessage('item added');
+            setMessage(formatMessage('item added'));
```

Comment:
Item added ?

Reply:
This is for the screen-reader, so it doesn't need capitalization.